docs/user/installation_and_configuration.rst

```diff
@@ -132,26 +132,37 @@ For Accumulo 1.6+
 
 Copying the runtime JAR to each tablet server as for Accumulo 1.5 above will
 still work, but in Accumulo 1.6, we can leverage namespaces to isolate the
-GeoMesa classpath from the rest of Accumulo. First, you have to create the
-namespace in the Accumulo shell:
+GeoMesa classpath from the rest of Accumulo.
+
+Execute the ``install-geomesa-namespace.sh`` script in the ``geomesa-$VERSION/dist/accumulo`` directory.
 
 .. code::
 
+    $ ./install-geomesa-namespace.sh -u root -n myNamespace
+
+The command line arguments the script accepts are:
+
+Required parameters:
+     -u (Accumulo username)
+     -n (Accumulo namespace)
+Optional parameters:
+     -p (Accumulo password)
+     -g (Path of GeoMesa distributed runtime JAR)
+     -h (HDFS URI e.g. hdfs://localhost:54310)
```

Comment:
Yeah, the script will try to find defaults for omitted optional params. Password will be prompted for.